ci/tasks/gcp/deploy-director-ipv6.sh

```
@@ -0,0 +1,75 @@
+director_name: stemcell-smoke-tests-director
+internal_cidr: ${INTERNAL_CIDR}
+internal_gw: ${INTERNAL_GW}
+network:    ${GCP_NETWORK_NAME}
+subnetwork: ${GCP_SUBNET_NAME}
+reserved_range: [${RESERVED_RANGE}]
+second_internal_cidr: ${SECOND_INTERNAL_CIDR}
+second_internal_gw: ${SECOND_INTERNAL_GW}
+second_internal_ip: ${SECOND_INTERNAL_IP}
+EOF
+
+echo ${GCP_JSON_KEY} > "${REPO_PARENT}/gcp_creds.json"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)